src/m365/spfx/commands/project/project-model/PackageSolutionJson.ts

```diff
@@ -10,6 +10,7 @@ export interface PackageSolutionJson extends JsonFile {
     metadata?: PackageSolutionJsonMetadata;
     skipFeatureDeployment?: boolean;
     version?: string;
+    name?: string;
```

Comment:
Let's sort properties alphabetically for easier lookup

Reply:
Good point. Will do that.